evals/configs/others/asyncFL.yml

```diff
@@ -0,0 +1,56 @@
+# Configuration file of FAR training experiment
+
+# ========== Cluster configuration ==========
+# ip address of the parameter server (need 1 GPU process)
+ps_ip: localhost
+
+# ip address of each worker:# of available gpus process on each gpu in this node
+# Note that if we collocate ps and worker on same GPU, then we need to decrease this number of available processes on that GPU by 1
+# E.g., master node has 4 available processes, then 1 for the ps, and worker should be set to: worker:3
+worker_ips:
+    - localhost:[2]
+
+exp_path: $FEDSCALE_HOME/fedscale/core
+
+# Entry function of executor and aggregator under $exp_path
+executor_entry: ../../fedscale_examples/Async_FL/executor.py
+
+aggregator_entry: ../../fedscale_examples/Async_FL/async_aggregator.py
```

Comment:
Is it possible to make this an optional config, in case the user wants to host a separate aggregation service. This would help decouple the client simulation from server.

Reply:
I think it works! we can just alternatively start this inherited aggregator/executor class by adding a new config.